apps/payload/src/payload.config.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable node/prefer-global/process */
+import { viteBundler } from '@payloadcms/bundler-vite'
+import { postgresAdapter } from '@payloadcms/db-postgres'
+import { slateEditor } from '@payloadcms/richtext-slate'
+import { buildConfig } from 'payload/config'
+import CurrentlyLearning from './collections/skills/currently-learning'
+import User from './collections/user/user'
+
+export default buildConfig({
+  serverURL: process.env.PAYLOAD_PUBLIC_EXTERNAL_SERVER_URL ?? '',
+  collections: [User, CurrentlyLearning],
+  admin: {
+    user: User.slug,
+    bundler: viteBundler(),
+  },
+  cors: process.env.WHITELIST_ORIGINS ? process.env.WHITELIST_ORIGINS.split(',') : [],
```

Comment:
Not sure if this is correct, while CORS and CSRF are different areas, it's only the same URLs that I'd allow to hit the payloadcms endpoints.

Reply:
Ah, I understand. The purpose of seperating them at the moment is to ensure that we have a separation of concerns so that down the line we can configure them more strictly? @sourcery-ai